src/components/FaceScreen.tsx

```diff
@@ -0,0 +1,69 @@
+import Box from '@mui/material/Box'
+import './../vars.ts'
+import React from 'react'
+import Bg from '../assets/pexels-alexandr-podvalny-1227513.jpeg'
+import { Stack, Typography } from '@mui/material'
+import CustomisedButton from './CustomisedButton.tsx'
+import { before } from 'node:test'
+
+interface FaceScreenProps {}
+
+const FaceScreen: React.FC<FaceScreenProps> = () => {
+  return (
+    <Box
+      component="div"
+      sx={{
+        position: 'relative',
+        backgroundImage: `url(${Bg})`,
+        backgroundColor: 'rgba(0,0,0,50%)',
+        backgroundSize: 'cover',
+        width: '100%',
+        maxWidth: '1170px',
+        height: '650px',
+        minHeight: '500px',
+        display: 'flex',
```

Comment:
Check if anything that you override via sx is not accessable from props for box component
Because appreciated to use its own props in first priority

Reply:
fixed